rfd/0001-mintlify-migration.md

```diff
@@ -0,0 +1,82 @@
+---
+authors: Paul Gottschling (paul.gottschling@goteleport.com), Walt Della (walt.della@goteleport.com)
+state: draft
+---
+
+# RFD 1 - Mintlify migration workflow
+
+We are moving the docs over to the hosted platform
+[Mintlify](https://mintlify.com/), and want to update both our Mintlify-managed
+site and our [current docs site](https://goteleport.com/docs/) with every
+documentation PR we merge to gravitational/teleport. This document outlines a
+high-level plan for managing the dual deployment.
+
+## How Mintlify works
```

Comment:
Can we add a section here about the Mintlify Github app, and how it should be configured.

Particularly, the Mintlify GitHub App will need to be installed on the teleport org, but we want to make sure:
- it doesn't have any org scopes
- it is limited to the "read content" repo scope
- the only allowed repo is `docs`